CONTRIBUTING.md

```diff
@@ -219,6 +219,45 @@ name.
 </record>
 ```
 
+### External dependencies
+
```

Comment:
I mean add something like:

#### README.rst
If your module use extras dependencies of python or binaries, please explain how to install them in the `README.rst` file in the section `Installation`.